lib/lita/handlers/github_pinger.rb

```diff
@@ -164,6 +164,11 @@ def act_on_pr_opened(body, response)
           puts "Sending DM to #{chosen_reviewer}..."
           send_dm(chosen_reviewer, message)
 
+          puts "Notifying PR owner of assignment..."
+          pr_owner = find_engineer(github: body["pull_request"]["user"]["login"])
+          assignment_message = "We assigned #{chosen_reviewer} to review #{body["pull_request"]["html_url"]}"
```

Comment:
what do you think about like, `"#{chosen_reviewer} has been notified via round-robin to review ${url}"`?